### PR TITLE
#5298: removed not working examples.

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
Removed not working examples or obsolete examples from MapStore.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#5298 . More examples was obsolete or not working

**What is the new behavior?**
Only 4 examples remained: 
2 core examples + 2 playgrounds (API usage example and featureEditor).

Can be replaced in the future with something more powerful like react storybook or some livecoding tool.

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: build/examples.js [hunk 1]
```diff
@@ -3,12 +3,6 @@ var path = require("path");
 module.exports = {
     mouseposition: path.join(__dirname, "..", "web", "client", "examples", "mouseposition", "app"),
     scalebar: path.join(__dirname, "..", "web", "client", "examples", "scalebar", "app"),
-    "3dviewer": path.join(__dirname, "..", "web", "client", "examples", "3dviewer", "app"),
     featuregrid: path.join(__dirname, "..", "web", "client", "examples", "featuregrid", "app"),
-    print: path.join(__dirname, "..", "web", "client", "examples", "print", "app"),
-    login: path.join(__dirname, "..", "web", "client", "examples", "login", "app"),
-    plugins: path.join(__dirname, "..", "web", "client", "examples", "plugins", "app"),
```
Comment (allyoucanmap) [inline build/examples.js:10]: API example uses the plugins example to store some information in the localStorage:

> Use the plugins example to save your configuration with a meaningful name and use it in the embedded example with the map=<name> parameter.

image

File: build/examples.js [hunk 2]
```diff
@@ -3,12 +3,6 @@ var path = require("path");
...
-    api: path.join(__dirname, "..", "web", "client", "examples", "api", "app"),
-    hooks: path.join(__dirname, "..", "web", "client", "examples", "hooks", "app"),
-    lazyplugins: path.join(__dirname, "..", "web", "client", "examples", "lazyplugins", "app")
```
Comment (allyoucanmap) [inline build/examples.js:13]: Probably we could remove `lazyplugins` and `hooks` from this index but should we remove also the related code? It's a new feature and we haven't so much reference to an App with lazy plugins or hooks but probably I'm wrong

File: build/examples.js (hunk 1 above)
Comment (offtherailz) [reply build/examples.js:10]: Ok, so I think we should remove also that functionality, thanks.

File: build/examples.js (hunk 2 above)
Comment (offtherailz) [reply build/examples.js:13]: I'm not sure they are maintained so I think we should provide an alternative and more powerful set of examples when framework has been revised, with something that is more useful for developers

**Review by allyoucanmap — APPROVED** [review]

**Comment by offtherailz** [comment]
Going to backport to 2020.02.xx

**Pull request merged**